README.adoc

```diff
@@ -80,7 +80,9 @@ messageId.subscribe(System.out::println);
 
 === Sending messages with cached producer
 
-By default a ConcurrentHashMap based cache is used. It's recommended to use a more advanced cache based on Caffeine. The cache will get used as the default implementation when it is on the classpath.
+By default, a ConcurrentHashMap based cache is used.
```

Comment:
In the spirit of Asciidoc's recommended [One Sentence Per Line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) I adjusted the existing doc a bit.